lib/you_track/client.rb

```diff
@@ -4,20 +4,8 @@ class YouTrack::Client < Cistern::Service
 
 require_relative "client/real"
 require_relative "client/mock"
-
 require_relative "client/model"
 require_relative "client/request"
-
 require_relative "client/login"
-require_relative "client/get_issue"
-require_relative "client/get_issues"
-require_relative "client/create_issue"
-require_relative "client/update_issue"
-require_relative "client/apply_issue_command"
-
-require_relative "client/get_issue_comments"
 
-require_relative "client/issue"
-require_relative "client/issues"
-require_relative "client/comment"
-require_relative "client/comments"
+Dir.glob(Dir.pwd + "/lib/you_track/client/{models,requests}/*.rb").each { |f| require_relative f }
```

Comment:
i'd stick to that for now. it's nice being explicit. and it's not much better then cistern 1.x `request, model, collection` stuff